ac_c.lua

```diff
@@ -72,11 +72,18 @@ Citizen.CreateThread(function()
 		if GetUsingnightvision() then
 			TriggerServerEvent("AntiCheese:Night")
 		end
-		
-		local playerPed = PlayerPedId()
```

Comment:
why?

Reply:
Yes, indeed, why would you unoptimise it by not localizing `playerPed`?